### PR TITLE
[v7r2] added Rucio agents to ConfigTemplate.cfg

### PR DESCRIPTION
**Pull request by fstagni**
(no need for release notes)

### CODE REVIEW

**Comment by andresailer** [comment]
Please also add them to https://github.com/DIRACGrid/DIRAC/blob/integration/src/DIRAC/ConfigurationSystem/test/Test_agentOptions.py

**Comment by fstagni** [comment]
The following integration test fails (fewer CFG locks):

https://github.com/DIRACGrid/DIRAC/pull/5194/checks?check_run_id=2820768579

```
=========================== 2 passed in 1.45 seconds ===========================
test_generateProxyDN (__main__.DIRACCAPPTest) ... ok
test_getProxy (__main__.DIRACCAPPTest) ... ERROR

======================================================================
ERROR: test_getProxy (__main__.DIRACCAPPTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/dirac/LocalRepo/TestCode/DIRAC/tests/Integration/Resources/ProxyProvider/Test_DIRACCAProxyProvider.py", line 89, in setUp
    self.assertTrue(result['OK'], '\n%s' % result.get('Message') or 'Error message is absent.')
AssertionError: 
getOptionsDict() failed with message: Path /Resources/ProxyProviders/DIRAC_TEST_CA does not exist or it's not a section
```

cc @atsareg @chrisburr

**Comment by chrisburr** [comment]
I've ran that test a few thousand times locally and haven't seen it fail again. Retrying on GitHub also fixed this issue so I'm inclined to make note but not worry for now.

File: .github/workflows/basic-python3.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ jobs:
           #   * `test_BaseType_Unicode` and `test_nestedStructure` fail due to
           #     DISET's string and unicode types being poorly defined
           - pytest --runslow -k 'not test_BaseType_Unicode and not test_nestedStructure'
-          - pylint -E src/
+          - pylint -E src/ --rcfile=.pylintrc
```
Comment (chrisburr) [inline .github/workflows/basic-python3.yml:23]: Isn't it the default?
```suggestion
          - pylint -E src/
```

Comment (chrisburr) [reply .github/workflows/basic-python3.yml:23]: If you want a hack for `pytest`, you can add a `git+https://` URL to the `pip` section of the `environment-py3.yaml` and add it to the list of commits to revert for #5193.

**Comment by chrisburr** [comment]
Readthedocs started failing as it insists on trying to upgrade all dependencies when installing DIRAC. M2Crypto has been updated on PyPI but not in conda-forge so it tries to update but as M2Crypto doesn't provide wheels on PyPI the installation fails. Apparently 8558130 is the preferred solution, see the commit message links for details.

**Pull request merged**